tests/test_device_tracker.py

```
@@ -0,0 +1,139 @@
+async def test_device_tracker_updates_with_coordinator(
+    hass: HomeAssistant, smart_fixture: respx.Router
+):
+    """
+    Test that device tracker updates when coordinator refreshes.
+
+    This test verifies that position data is updated when the coordinator
+    fetches new data from the API.
+    """
+    position_values = {"latitude": "123456789", "longitude": "987654321"}
+
+    async def update_position(request: Request, route: respx.Route) -> Response:
```

Ruff (0.14.6)

Unused function argument: `request`

(ARG001)